internal/provider/serverless_cluster.go

```diff
@@ -54,17 +56,57 @@ func ResourceServerlessCluster() *schema.Resource {
 				// always returns a non-nil error.
 				// ForceNew: true,
 				Type: schema.TypeString,
+				// TODO: Make ForceNew to work
+				// ForceNew:    true,
 			},
 			"free_tier": {
 				Description: "Whether a cluster is in a free tier",
-				Computed:    true,
+				Default:     true,
+				Type:        schema.TypeBool,
+				Optional:    true,
+				// TODO: Make ForceNew to work
+				// ForceNew:    true,
+			},
+			"enable_dns": {
+				Description: "Whether to enable CNAME for seed nodes",
+				Optional:    true,
 				Type:        schema.TypeBool,
+				// TODO: Make ForceNew to work
+				// ForceNew:    true,
+				Default: true,
+			},
+			"units": {
+				Description: "Processing units",
+				Default:     1,
+				Type:        schema.TypeInt,
+				Optional:    true,
+				// TODO: Make ForceNew to work
+				// ForceNew:    true,
+			},
+			"hours": {
+				Description: "Hours for cluster to last",
+				Type:        schema.TypeInt,
+				Optional:    true,
+				// TODO: Make ForceNew to work
+				// ForceNew:    true,
```

Comment:
Only one ForceNew should be commented out - rest should be uncommented (so we maximize the correctness/accuracy in the initial attr validation)

Reply:
done

Reply:
I see it's still commented out here

Reply:
done